.github/workflows/maven.yml

```
@@ -205,4 +206,11 @@ jobs:
             all-in-one-apim/modules/integration-v2/tests-integration/cucumber-tests/target/surefire-reports/**
             all-in-one-apim/modules/integration-v2/tests-integration/cucumber-tests/target/cucumber-report/**
             all-in-one-apim/modules/integration-v2/tests-integration/cucumber-tests/target/logs/**
+            all-in-one-apim/modules/integration-v2/tests-integration/cucumber-tests/target/coverage/output/**
           if-no-files-found: warn
+      - name: Upload v2 integration coverage to Codecov
+        if: always()
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)